pkg/apis/options/options.go

```diff
@@ -37,6 +37,7 @@ type Options struct {
 
 	AuthenticatedEmailsFile  string   `flag:"authenticated-emails-file" cfg:"authenticated_emails_file"`
 	KeycloakGroup            string   `flag:"keycloak-group" cfg:"keycloak_group"`
+	KeycloakRoles            []string `flag:"keycloak-roles" cfg:"keycloak_roles"`
```

Comment:
```suggestion
	KeycloakRoles            []string `flag:"keycloak-role" cfg:"keycloak_roles"`
```

Flags are singular, configs are plural in multi-param cases.